test/specs/shape/SymbolsSpec.js

```diff
@@ -1,36 +1,60 @@
 import React from 'react';
 import { expect } from 'chai';
-import { Surface, Symbols } from 'recharts';
-import { mount, render } from 'enzyme';
+import { Surface, Symbols, LineChart, XAxis, Line, YAxis } from 'recharts';
+import { render as enzymeRender } from 'enzyme';
+import { render } from '@testing-library/react';
 
 describe('<Symbols />', () => {
   it('Render 1 symbol', () => {
-    const wrapper = render(
+    const wrapper = enzymeRender(
       <Surface width={400} height={400}>
         <Symbols cx={100} cy={100} size={100} fill="#ff7300" />
-      </Surface>
+      </Surface>,
     );
 
     expect(wrapper.find('.recharts-symbols').length).to.equal(1);
   });
 
   it('Render 1 symbol when type is wrong', () => {
-    const wrapper = render(
+    const wrapper = enzymeRender(
       <Surface width={400} height={400}>
         <Symbols cx={100} cy={100} size={100} type="anything" fill="#ff7300" />
-      </Surface>
+      </Surface>,
     );
 
     expect(wrapper.find('.recharts-symbols').length).to.equal(1);
   });
 
   it("Don't render any symbol when cx is invalid", () => {
-    const wrapper = render(
+    const wrapper = enzymeRender(
       <Surface width={400} height={400}>
         <Symbols cy={100} size={100} fill="#ff7300" />
-      </Surface>
+      </Surface>,
     );
 
     expect(wrapper.find('.recharts-symbols').length).to.equal(0);
   });
+
+  it('Expect no points attribute on path', () => {
+    const { container } = render(
+      <LineChart
```

Comment:
Do you need a full LineChart for this test? 
The other tests just use a Surface and Symbols, 
could you not use a line directly? 

The fact that this test is different in setup makes me wonder wether it belongs in a separate file?

Reply:
This is where the issue occurs (since this is where `filterProps` is called). This test asserts something from the element that is being tested. I wouldn't want to put it in `LineChart` since the issue doesn't come from line chart. If I do not render a chart then there is no issue with `Surface` (no "points" spread)